src/events.zig

```diff
@@ -536,6 +639,62 @@ pub const Event = union(Type) {
                     .repeat = event.key.repeat,
                 },
             },
+            C.SDL_EVENT_MOUSE_MOTION => .{ .mouse_motion = .{
+                .common = .fromSdl(&event),
+                .window_id = if (event.motion.windowID == 0) null else event.motion.windowID,
+                .which = .{
```

Comment:
If `which` is nullable, then shouldn't we check for `0` as a special case to assign `null`?